src/vue/components/inflections-table-wide.vue

```diff
@@ -243,8 +290,10 @@ export default {
   },
 
   watch: {
-    view: function () {
-      this.state.noSuffixGroupsHidden = this.view.isNoSuffixMatchesGroupsHidden
+    view: function (view) {
```

Comment:
May be it is better not to watch by such a complex object as a view, may be it would be better to create some scalar variable to track changes?

Reply:
This is a good point. Adding one more variable adds complexity, but we could have some performance gains in return. I'll think about if we can have some good compromise between the two.

Reply:
We were talking before about such optimizations here
https://github.com/alpheios-project/components/issues/365

And I used computed property for complex objects, because it is really re-checks only when some variable inside is changed. 

So in wordListPanel I previously used local scalar property updateTime, for now it is Vuex store property - wordListUpdateTime.
Such an approach could help eliminate such complex rechecking inside watch on every update step.

You could see it for example in `word-language-panel.vue` - in `wordItems` computed property.

Reply:
Here it is a useful discussion about that - https://stackoverflow.com/questions/43277705/vuejs-difference-between-computed-property-and-watcher
Hope, it would be interesting!

Reply:
I think with some reshuffling of the logic we can eliminate the use of the `view` watcher completely. I will implement it and test now.

Reply:
It seems to be really hard to go without a watcher here. But I replaced a watcher of the whole View object with the watcher of the View.id (a string). It will allow to avoid performance penalty for watching a large object yet will keep things simple as it will require no additional tracking variable.

Reply:
I think it is a good use of watch!